requirements-dev.txt

```diff
@@ -1,7 +1,8 @@
-black==22.10.0
-flake8==6.0.0
-tiled[all]
+black==24.2.0
+flake8==7.0.0
+tiled[all]==0.1.0a114
```

Comment:
Do we need `tiled[all]` in this project? Would `tiled[client]` be sufficient?

Reply:
That's a good point, with a more complete test-setup in [mlexchange/mlex_tomo_framework](https://github.com/mlexchange/mlex_tomo_framework), it makes sense to remove this here.